pom.xml

```diff
@@ -153,6 +153,7 @@
                 <versionOsgiRule implementation="org.eclipse.jetty.toolchain.enforcer.rules.RequireOsgiCompatibleVersionRule" />
                 <versionRedhatRule implementation="org.eclipse.jetty.toolchain.enforcer.rules.RequireRedhatCompatibleVersionRule" />
                 <versionDebianRule implementation="org.eclipse.jetty.toolchain.enforcer.rules.RequireDebianCompatibleVersionRule" />
+                <requireUpperBoundDeps/>
```

Comment:
So this is the heart of the change?  the rest is just resolving issues that it created?

Reply:
yes it is so simple :) I would say resolving the "issues finally discovered" :P